webpack/components/extensions/Hosts/BulkActions/BulkErrataWizard/02_BulkErrataTable.js

```diff
@@ -0,0 +1,172 @@
+import React, { useEffect, useContext } from 'react';
+import {
+  Alert,
+  ToolbarItem,
+  Text,
+  TextContent,
+  TextVariants,
+} from '@patternfly/react-core';
+import { TableText } from '@patternfly/react-table';
+import TableIndexPage from 'foremanReact/components/PF4/TableIndexPage/TableIndexPage';
+import { translate as __ } from 'foremanReact/common/I18n';
+import SelectAllCheckbox from 'foremanReact/components/PF4/TableIndexPage/Table/SelectAllCheckbox';
+import { STATUS, getControllerSearchProps } from 'foremanReact/constants';
+import { noop } from 'foremanReact/common/helpers';
+import { RowSelectTd } from 'foremanReact/components/HostsIndex/RowSelectTd';
+import { getPageStats } from 'foremanReact/components/PF4/TableIndexPage/Table/helpers';
+import { BulkErrataWizardContext, getErrataUrl } from './BulkErrataWizard';
+import { ErrataType, ErrataSeverity } from '../../../../../components/Errata';
+import katelloApi from '../../../../../services/api';
+
+const BulkErrataTable = () => {
+  const {
+    setShouldValidateStep2,
+    errataBulkSelect,
+    errataResults: results,
+    errataMetadata: {
+      total, per_page: perPage, page, subtotal,
+    },
+    errataResponse: response,
+  } = useContext(BulkErrataWizardContext);
+  const ERRATA_URL = getErrataUrl();
+  const apiOptions = { key: 'BULK_HOST_ERRATA' };
+  const {
+    status: errataStatus,
+  } = response;
+
+
+  const origSearchProps = getControllerSearchProps('errata', 'searchBar-errata');
+  const customSearchProps = {
+    ...origSearchProps,
+    autocomplete: {
+      ...origSearchProps.autocomplete,
+      url: katelloApi.getApiUrl('/errata/auto_complete_search'),
+    },
+  };
+
+  const {
+    selectPage,
+    selectNone,
+    selectOne,
+    isSelected,
+    selectedCount,
+    areAllRowsSelected,
+    areAllRowsOnPageSelected,
+    updateSearchQuery,
+    hasInteracted,
+  } = errataBulkSelect;
+
+  useEffect(() => {
+    if (results?.length && hasInteracted) {
+      setShouldValidateStep2(true);
+    }
+  }, [setShouldValidateStep2, results, hasInteracted]);
+
+  const pageStats = getPageStats({ total: subtotal, page, perPage });
+  const selectionToolbar = (
+    <ToolbarItem key="selectAll">
+      <SelectAllCheckbox
+        {...{
+          selectNone,
+          selectPage,
+          selectedCount,
+          pageRowCount: pageStats.pageRowCount,
+          areAllRowsSelected,
+          areAllRowsOnPageSelected,
+        }}
+        selectAll={noop}
```

Comment:
I think select all should be enabled for this one. No harm in applying all possible errata.